app/components/elements/forms/label_component.html.erb

```diff
@@ -1 +1,4 @@
-  <%= form.label field_name, label_text, class: classes %>
+<%= form.label field_name, label_text, class: classes %>
+<% if render_tooltip? %>
+  <%= helpers.info_icon(fill: true, classes: 'px-3 tooltip-info', data: { bs_custom_class: 'tooltip-shift-right', bs_toggle: 'tooltip', bs_title: tooltip, bs_trigger: 'click focus', tooltips_target: 'icon' }) %>
```

Comment:
Componentize so that can be used elsewhere?

Reply:
I was under the assumption that we had tooltips all over the place, not just in form labels. If that is mistaken, then agree that no need to componentize.

Reply:
Other than the collection and work forms, there are also tool tips in the help form. See #670